styles/mobile..css

```diff
@@ -100,3 +101,386 @@ ul#horizontal-list li {
   display: inline;
   margin: 0 10px;
 }
+
+/* STARTING THE NEW SECTIONS */
+
+.middle-section {
+  display: grid;
+  height: 100vh;
+  background-color: #e5e5e5;
+}
+
+.mrw {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  padding-top: 120px;
+  padding-right: 24px;
+  padding-bottom: 40px;
+  padding-left: 36px;
+}
+
+.mid-border {
+  padding-left: 36px;
+  width: 327px;
+}
+
+.feature-image {
+  padding-right: 24px;
+  padding-top: 12px;
+  padding-left: 24px;
+}
+
+.img1 {
+  padding-top: 80px;
+}
+
+.mps {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  padding-top: 24px;
+}
+
+.description2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  width: 327px;
+  height: 147px;
+}
+
+.wrapper1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 12px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.box1 {
+  color: #091e42;
+  padding: 10px 12px 10px 12px;
+  border: 1px solid #8993a4;
+}
+
+.see-project-btn {
+  padding-top: 12px;
+}
+
+.btn1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #ff6b00;
+  width: 124px;
+  height: 48px;
+  border: none;
+  color: white;
+}
+
+/* Card  Starts Here */
+.cards-details {
+  margin-top: 24px;
+  padding-left: 24px;
+}
+
+.card-container {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url(../images/card-img.png);
+  height: 386px;
+  width: 327px;
+  padding-left: 16px;
+}
+
+.title2 {
+  color: white;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  padding-top: 84px;
+}
+
+.description3 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  width: 327px;
+  height: 87px;
+  padding-top: 12px;
+}
+
+.box2 {
+  color: #fff;
+  padding: 10px 12px 10px 12px;
+  border: 1px solid #8993a4;
+  padding-top: 12px;
+}
+
+.btn2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #ff6b00;
+  width: 343px;
+  height: 48px;
+  border: none;
+  color: white;
+}
+
+.empty-divider {
+  padding-top: 80px;
+}
+
+/* ABOUT ME */
+.aboutme {
+  height: 100vh;
+  background-color: #1c1a19;
+  background-image: url(../images/illustration-about-me-mobile.png);
+  background-position: right bottom;
+  background-size: 217px;
+  background-repeat: no-repeat;
```

Comment:
- [ ] This is great work and really deserves all the praise. Howver I would kindly suggest that you use flexbox to layout the ```about me``` since it is in the project requirement. The purpose of this is to enable you to as much as possible, get used to the flexbox property. 
![image](https://user-images.githubusercontent.com/107795045/219814434-0180e66d-2a67-4a48-9f6a-ffc82a218b3d.png)